### PR TITLE
Use gh cli instead of tkn task

### PR DESCRIPTION
**Pull request by furiosamg**
Tekton task를 이용해 댓글을 다는 것 대신 gh cli를 직접 사용해 댓글을 달게 변경했습니다.
pod을 새로 만들어도 되지 않아도 되서 빨라지는 장점이 있고, 관리할 데이터가 줄어 조금 편해집니다! ㅎㅎㅎ
또한 regression test 전에 pip freeze 결과를 댓글로 달도록 변경했습니다. (추후 정확도 재현을 위해)

~CI를 통과하기 위해 c05386354f96d655ecf56d857d8654b25f495a0d 커밋을 임시로 넣어 두었는데, #151 이 메인에 머지되고 나면 빼겠습니다. 그러면서 commit 목록도 좀 정리하겠습니다.~ 했습니다.

~또한 regression test ci 말고 본 ci 중 test-examples 가 실패하는 이유는 https://furiosa-ai.slack.com/archives/GRZ8J7X8U/p1687948029380609 때문이며, #151  이 머지되면 해결될 것으로 기대됩니다.~ ci 이제 성공합니다.

### CODE REVIEW

**Comment by furiosa-infra** [comment]
## Pip freeze result for ssd-mobilenet
```
aiobotocore==2.5.1
aiofiles==22.1.0
aiohttp==3.8.4
aiohttp-retry==2.8.3
aioitertools==0.11.0
aiosignal==1.3.1
amqp==5.1.1
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
async-timeout==4.0.2
asyncssh==2.13.2
atpublic==4.0
attrs==23.1.0
billiard==4.1.0
black==23.3.0
boto3==1.26.161
botocore==1.29.161
celery==5.3.1
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.3
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.3.0
colorama==0.4.6
configobj==5.0.8
contourpy==1.1.0
cryptography==41.0.1
cycler==0.11.0
Cython==0.29.35
dictdiffer==0.9.0
diskcache==5.6.1
distro==1.8.0
dpath==2.1.6
dulwich==0.21.5
dvc==3.2.2
dvc-data==2.3.1
dvc-http==2.30.2
dvc-objects==0.23.0
dvc-render==0.5.3
dvc-s3==2.23.0
dvc-studio-client==0.10.0
dvc-task==0.3.0
exceptiongroup==1.1.1
filelock==3.12.2
flatten-dict==0.4.2
flufl-lock==8.0.1
fonttools==4.40.0
frozenlist==1.3.3
fsspec==2023.6.0
funcy==2.0
furiosa-common==0.9.1
furiosa-models @ file:///app
furiosa-native-postprocess==0.9.0.dev0
furiosa-registry==0.9.1
furiosa-runtime==0.9.1
gitdb==4.0.10
GitPython==3.1.31
grandalf==0.8
hydra-core==1.3.2
idna==3.4
importlib-resources==5.12.0
iniconfig==2.0.0
isort==5.12.0
iterative-telemetry==0.0.8
jmespath==1.0.1
kiwisolver==1.4.4
kombu==5.3.1
markdown-it-py==3.0.0
matplotlib==3.7.1
mdurl==0.1.2
multidict==6.0.4
multipledispatch==1.0.0
mypy-extensions==1.0.0
nanotime==0.5.2
networkx==3.1
numpy==1.25.0
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
omegaconf==2.3.0
opencv-python-headless==4.7.0.72
orjson==3.9.1
packaging==23.1
pandas==1.5.3
pathspec==0.11.1
Pillow==9.5.0
platformdirs==3.8.0
pluggy==1.2.0
prompt-toolkit==3.0.38
psutil==5.9.5
py-cpuinfo==9.0.0
pycocotools==2.0.6
pycparser==2.21
pydantic==1.10.9
pydot==1.4.2
pygit2==1.12.2
Pygments==2.15.1
pygtrie==2.5.0
pyparsing==3.1.0
pytest==7.4.0
pytest-asyncio==0.17.2
pytest-benchmark==4.0.0
python-dateutil==2.8.2
pytz==2023.3
PyYAML==6.0
requests==2.31.0
rich==13.4.2
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
s3fs==2023.6.0
s3transfer==0.6.1
scmrepo==1.0.4
shortuuid==1.0.11
shtab==1.6.2
six==1.16.0
smmap==5.0.0
sqltrie==0.7.0
tabulate==0.9.0
toml==0.10.2
tomli==2.0.1
tomlkit==0.11.8
torch==1.13.1
torchvision==0.14.1
tqdm==4.65.0
typing_extensions==4.7.0
tzdata==2023.3
urllib3==1.26.16
vine==5.0.0
voluptuous==0.13.1
wcwidth==0.2.6
wrapt==1.15.0
yarl==1.9.2
zc.lockfile==3.0.post1
zipp==3.15.0
```

**Comment by furiosa-infra** [comment]
f9de735 ci: don't separate report step
```

----------------------------------------------------------------------------------------------------- benchmark: 2 tests ----------------------------------------------------------------------------------------------------
Name (time in ms)                                                Min                 Max               Mean            StdDev             Median               IQR            Outliers      OPS            Rounds  Iterations
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_mlcommons_ssd_mobilenet_with_native_rust_pp_accuracy     5.3836 (1.0)       26.0960 (1.0)      12.2434 (1.0)      3.6026 (1.0)      12.8739 (1.0)      7.1459 (1.89)       2449;0  81.6765 (1.0)        5000           1
test_mlcommons_ssd_mobilenet_accuracy                         7.7584 (1.44)     206.1524 (7.90)     16.9385 (1.38)     4.3591 (1.21)     18.0736 (1.40)     3.7748 (1.0)      1039;140  59.0371 (0.72)       5000           1
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
```

File: .ci/furiosa-models-regression-test/regression-test.yaml [hunk 1]
```diff
@@ -34,18 +46,36 @@ spec:
...
-        set -e
-        set -x
+        set -ex
+        git config --global --add safe.directory /workspace/source
+
+        echo -n "## Pip freeze result for " > pip_freeze.txt
+        echo $(params.modelName) >> pip_freeze.txt
+        echo "\`\`\`" >> pip_freeze.txt
+        pip freeze >> pip_freeze.txt
+        echo "\`\`\`" >> pip_freeze.txt
```
Comment (sukyoungjeong-furiosa) [inline .ci/furiosa-models-regression-test/regression-test.yaml:65]: https://stackoverflow.com/questions/28607382/is-it-possible-to-create-a-toggle-switch-in-markdown 참고해서 toggle 넣으면 어떨까 싶네요.

**Review by sukyoungjeong-furiosa — APPROVED** [review]

File: .ci/furiosa-models-regression-test/regression-test.yaml [hunk 2]
```diff
@@ -34,18 +46,36 @@ spec:
       env:
         - name: "PIP_EXTRA_INDEX_URL"
           value: "https://internal-pypi.furiosa.dev/simple"
+        - name: "GITHUBi_TOKEN"
```
[the remark this answers is not in the document]
Comment (furiosamg) [reply .ci/furiosa-models-regression-test/regression-test.yaml:49]: 앗 리베이스하다 실수했네요 감사합니다!

**Comment by furiosa-infra** [comment]
## Pip freeze result for ssd-mobilenet
<details>\n    <summary>pip freeze result</summary>
```
aiobotocore==2.5.1
aiofiles==22.1.0
aiohttp==3.8.4
aiohttp-retry==2.8.3
aioitertools==0.11.0
aiosignal==1.3.1
amqp==5.1.1
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
async-timeout==4.0.2
asyncssh==2.13.2
atpublic==4.0
attrs==23.1.0
billiard==4.1.0
black==23.3.0
boto3==1.26.161
botocore==1.29.161
celery==5.3.1
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.3
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.3.0
colorama==0.4.6
configobj==5.0.8
contourpy==1.1.0
cryptography==41.0.1
cycler==0.11.0
Cython==0.29.35
dictdiffer==0.9.0
diskcache==5.6.1
distro==1.8.0
dpath==2.1.6
dulwich==0.21.5
dvc==3.2.2
dvc-data==2.3.1
dvc-http==2.30.2
dvc-objects==0.23.0
dvc-render==0.5.3
dvc-s3==2.23.0
dvc-studio-client==0.10.0
dvc-task==0.3.0
exceptiongroup==1.1.1
filelock==3.12.2
flatten-dict==0.4.2
flufl-lock==8.0.1
fonttools==4.40.0
frozenlist==1.3.3
fsspec==2023.6.0
funcy==2.0
furiosa-common==0.9.1
furiosa-models @ file:///app
furiosa-native-postprocess==0.9.0.dev0
furiosa-registry==0.9.1
furiosa-runtime==0.9.1
gitdb==4.0.10
GitPython==3.1.31
grandalf==0.8
hydra-core==1.3.2
idna==3.4
importlib-resources==5.12.0
iniconfig==2.0.0
isort==5.12.0
iterative-telemetry==0.0.8
jmespath==1.0.1
kiwisolver==1.4.4
kombu==5.3.1
markdown-it-py==3.0.0
matplotlib==3.7.1
mdurl==0.1.2
multidict==6.0.4
multipledispatch==1.0.0
mypy-extensions==1.0.0
nanotime==0.5.2
networkx==3.1
numpy==1.25.0
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
omegaconf==2.3.0
opencv-python-headless==4.7.0.72
orjson==3.9.1
packaging==23.1
pandas==1.5.3
pathspec==0.11.1
Pillow==9.5.0
platformdirs==3.8.0
pluggy==1.2.0
prompt-toolkit==3.0.38
psutil==5.9.5
py-cpuinfo==9.0.0
pycocotools==2.0.6
pycparser==2.21
pydantic==1.10.9
pydot==1.4.2
pygit2==1.12.2
Pygments==2.15.1
pygtrie==2.5.0
pyparsing==3.1.0
pytest==7.4.0
pytest-asyncio==0.17.2
pytest-benchmark==4.0.0
python-dateutil==2.8.2
pytz==2023.3
PyYAML==6.0
requests==2.31.0
rich==13.4.2
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
s3fs==2023.6.0
s3transfer==0.6.1
scmrepo==1.0.4
shortuuid==1.0.11
shtab==1.6.2
six==1.16.0
smmap==5.0.0
sqltrie==0.7.0
tabulate==0.9.0
toml==0.10.2
tomli==2.0.1
tomlkit==0.11.8
torch==1.13.1
torchvision==0.14.1
tqdm==4.65.0
typing_extensions==4.7.0
tzdata==2023.3
urllib3==1.26.16
vine==5.0.0
voluptuous==0.13.1
wcwidth==0.2.6
wrapt==1.15.0
yarl==1.9.2
zc.lockfile==3.0.post1
zipp==3.15.0
```
</details>

**Comment by furiosa-infra** [comment]
f9de735 ci: don't separate report step
```

----------------------------------------------------------------------------------------------------- benchmark: 2 tests ----------------------------------------------------------------------------------------------------
Name (time in ms)                                                Min                 Max               Mean            StdDev             Median               IQR            Outliers      OPS            Rounds  Iterations
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
test_mlcommons_ssd_mobilenet_with_native_rust_pp_accuracy     5.9723 (1.0)       23.9658 (1.0)      12.3947 (1.0)      3.7083 (1.02)     13.1696 (1.0)      7.3626 (3.69)       2453;0  80.6799 (1.0)        5000           1
test_mlcommons_ssd_mobilenet_accuracy                         8.4099 (1.41)     219.4757 (9.16)     18.5882 (1.50)     3.6437 (1.0)      18.8166 (1.43)     1.9952 (1.0)       489;426  53.7976 (0.67)       5000           1
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
```

**Comment by furiosa-infra** [comment]
## Pip freeze result for ssd-mobilenet
<details><summary>pip freeze result</summary><pre>
aiobotocore==2.5.1
aiofiles==22.1.0
aiohttp==3.8.4
aiohttp-retry==2.8.3
aioitertools==0.11.0
aiosignal==1.3.1
amqp==5.1.1
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
async-timeout==4.0.2
asyncssh==2.13.2
atpublic==4.0
attrs==23.1.0
billiard==4.1.0
black==23.3.0
boto3==1.26.161
botocore==1.29.161
celery==5.3.1
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.3
click-didyoumean==0.3.0
click-plugins==1.1.1
click-repl==0.3.0
colorama==0.4.6
configobj==5.0.8
contourpy==1.1.0
cryptography==41.0.1
cycler==0.11.0
Cython==0.29.35
dictdiffer==0.9.0
diskcache==5.6.1
distro==1.8.0
dpath==2.1.6
dulwich==0.21.5
dvc==3.2.2
dvc-data==2.3.1
dvc-http==2.30.2
dvc-objects==0.23.0
dvc-render==0.5.3
dvc-s3==2.23.0
dvc-studio-client==0.10.0
dvc-task==0.3.0
exceptiongroup==1.1.1
filelock==3.12.2
flatten-dict==0.4.2
flufl-lock==8.0.1
fonttools==4.40.0
frozenlist==1.3.3
fsspec==2023.6.0
funcy==2.0
furiosa-common==0.9.1
furiosa-models @ file:///app
furiosa-native-postprocess==0.9.0.dev0
furiosa-registry==0.9.1
furiosa-runtime==0.9.1
gitdb==4.0.10
GitPython==3.1.31
grandalf==0.8
hydra-core==1.3.2
idna==3.4
importlib-resources==5.12.0
iniconfig==2.0.0
isort==5.12.0
iterative-telemetry==0.0.8
jmespath==1.0.1
kiwisolver==1.4.4
kombu==5.3.1
markdown-it-py==3.0.0
matplotlib==3.7.1
mdurl==0.1.2
multidict==6.0.4
multipledispatch==1.0.0
mypy-extensions==1.0.0
nanotime==0.5.2
networkx==3.1
numpy==1.25.0
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
omegaconf==2.3.0
opencv-python-headless==4.7.0.72
orjson==3.9.1
packaging==23.1
pandas==1.5.3
pathspec==0.11.1
Pillow==9.5.0
platformdirs==3.8.0
pluggy==1.2.0
prompt-toolkit==3.0.38
psutil==5.9.5
py-cpuinfo==9.0.0
pycocotools==2.0.6
pycparser==2.21
pydantic==1.10.9
pydot==1.4.2
pygit2==1.12.2
Pygments==2.15.1
pygtrie==2.5.0
pyparsing==3.1.0
pytest==7.4.0
pytest-asyncio==0.17.2
pytest-benchmark==4.0.0
python-dateutil==2.8.2
pytz==2023.3
PyYAML==6.0
requests==2.31.0
rich==13.4.2
ruamel.yaml==0.17.32
ruamel.yaml.clib==0.2.7
s3fs==2023.6.0
s3transfer==0.6.1
scmrepo==1.0.4
shortuuid==1.0.11
shtab==1.6.2
six==1.16.0
smmap==5.0.0
sqltrie==0.7.0
tabulate==0.9.0
toml==0.10.2
tomli==2.0.1
tomlkit==0.11.8
torch==1.13.1
torchvision==0.14.1
tqdm==4.65.0
typing_extensions==4.7.0
tzdata==2023.3
urllib3==1.26.16
vine==5.0.0
voluptuous==0.13.1
wcwidth==0.2.6
wrapt==1.15.0
yarl==1.9.2
zc.lockfile==3.0.post1
zipp==3.15.0
</pre></details>

File: .ci/furiosa-models-regression-test/regression-test.yaml (hunk 1 above)
Comment (furiosamg) [reply .ci/furiosa-models-regression-test/regression-test.yaml:66]: https://github.com/furiosa-ai/furiosa-models/pull/154#issuecomment-1612583917
토글 기능 들어갔습니다~

File: .ci/furiosa-models-regression-test/regression-test.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (furiosamg) [reply .ci/furiosa-models-regression-test/regression-test.yaml]: https://github.com/furiosa-ai/furiosa-models/compare/a011dfac1ce3149f57cd56ad614cf2cc6aaea861..82f022b9821ad495162ae641c189e67360131cc2
에 반영했습니다

**Comment by sukyoungjeong-furiosa** [comment]
수고하셨습니다. 머지할게요.

**Pull request merged**